### ISSUE TITLE
link error: relocation ".uw_base+0x0 (type R_X86_64_RELATIVE)" goes out of range

### ISSUE THREAD

**Issue by lichn666** [vlang/v#8491]
When using two packages that call C function, for example ：
```
import sqlite
import json
......
an error is returned:
==================
C:/Program Files/mingw-w64/x86_64-8.1.0-win32-sjlj-rt_v6-rev0/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:\Code\vFile\tempCodeRunnerFile.exe(.pdata): relocation ".uw_base+0x0 (type R_X86_64_RELATIVE)" goes out of range
collect2.exe: error: ld returned 1 exit status
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.
```

**Comment by JalonSolov** [comment]
First suggestion:  Update to newer version of gcc.

You can download v10.2 of gcc from http://winlibs.com/

I'm not sure if it will resolve this particular problem, but at least you'll have the latest version of gcc.

**Comment by lichn666** [comment]
My environment：
windows10
MinGW-w64 8.1.0 
v weekly.2021.5

**Comment by spaceface777** [comment]
Fixed in vlang/tccbin@bac32fd.
